sprints/dashboard/utils.py

```diff
@@ -97,6 +101,67 @@ def get_cell_member_names(conn: CustomJira, members: Iterable[str]) -> Dict[str,
     """Returns cell members with their names."""
     return {conn.user(member).displayName: member for member in members}
 
```

Comment:
There should be two lines of space between these functions. Not sure why `pylint` didn't catch this…
Edit. ah, I see that we had to disable it because of some compatibility problems. Therefore please be extra careful about PEP-8…